sdk/search/search/review/search.api.md

```diff
@@ -10,6 +10,48 @@ import { PipelineOptions } from '@azure/core-http';
 import { ServiceClientCredentials } from '@azure/core-http';
 import { WebResource } from '@azure/core-http';
 
+// @public
+export interface AccessCondition {
```

Comment:
@richardpark-msft for Appconfig, was the decision to flatten these onto the options important? Or is it OK to have an options subkey for it?